internal/backends/qt/qt_widgets.rs

```diff
@@ -89,8 +90,9 @@ macro_rules! fn_render {
                 backend.draw_cached_pixmap(
                     item_rc,
                     &|callback| {
-                        let width = self.width().get() * $dpr;
-                        let height = self.height().get() * $dpr;
+                        let geo = item_rc.geometry();
```

Comment:
Ideally here we should get the size from the call to render that has it instead of doing that.
(because each call to geometry add possibly 4 property dependency we don't need)

But we can leave that as an action item for later.